### PR TITLE
Morsel CONTAINS, range, composite index stuff

### PR DESCRIPTION
**Pull request by sherfert**
...

### CODE REVIEW

File: enterprise/cypher/acceptance-spec-suite/src/test/scala/org/neo4j/internal/cypher/acceptance/MorselRuntimeAcceptanceTest.scala [hunk 1]
```diff
@@ -275,7 +274,58 @@ class MorselRuntimeAcceptanceTest extends ExecutionEngineFunSuite {
     val resultSet = asScalaResult(result).toSet
     resultSet.map(map => map("n.name")) should equal(Set("Satia42"))
     result.getExecutionPlanDescription.getArguments.get("runtime") should equal("MORSEL")
+  }
```
Comment (systay) [inline enterprise/cypher/acceptance-spec-suite/src/test/scala/org/neo4j/internal/cypher/acceptance/MorselRuntimeAcceptanceTest.scala:304]: This whole class should be rewritten to use CypherComparisonSupport. We should not add more tests to this file, IMHO...

File: enterprise/cypher/morsel-runtime/src/main/scala/org/neo4j/cypher/internal/runtime/vectorized/operators/NodeIndexContainsScanOperator.scala [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+  override def operate(message: Message,
+                       data: Morsel,
+                       context: QueryContext,
+                       state: QueryState): Continuation = {
+    var valueIndexCursor: NodeValueIndexCursor  = null
+    var iterationState: Iteration = null
+    val read = context.transactionalContext.dataRead
+    val index = context.transactionalContext.schemaRead.index(label, propertyKey)
+
+    var exprIsString: Boolean = true
```
Comment (systay) [inline enterprise/cypher/morsel-runtime/src/main/scala/org/neo4j/cypher/internal/runtime/vectorized/operators/NodeIndexContainsScanOperator.scala]: This is not so much about type, but more about `NULL`s, right? Other types are handled by throwing an exception. Maybe call the variable something with null instead?

**Review by systay — APPROVED** [review]
Looks fine. We should fix the testing, but maybe that can come later

**Comment by sherfert** [comment]
CypherComparisonSupport was disabled for morsel because it kept hanging. Pontus and I fixed that today, so we can tackle this test class soon in another PR.

**Pull request merged**